package.json

```diff
@@ -43,20 +43,19 @@
   "devDependencies": {
     "@fastify/swagger": "^8.14.0",
     "@fastify/swagger-ui": "^3.0.0",
-    "@types/jest": "^29.5.12",
     "@types/node": "^20.12.12",
     "@typescript-eslint/eslint-plugin": "^7.9.0",
     "@typescript-eslint/parser": "^7.9.0",
+    "@vitest/coverage-v8": "^1.6.0",
```

Comment:
jest matchInlineSnapshot is no longer compatible with Prettier, so had to switch to vitest